docs/supported_apis/dataframe_supported.rst

```diff
@@ -185,7 +185,7 @@ default to pandas.
 | ``iloc``                   | `iloc`_                   | Y                      | **Omnisci**: ``P``, read access fully supported,   |
 |                            |                           |                        | write access: no row and 2D assignments support    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``infer_objects``          | `infer_objects`_          | D                      |                                                    |
+| ``infer_objects``          | `infer_objects`_          | Y                      |                                                    |
```

Comment:
that change is questionable, I think it should be `P` for `Partial`, unless you say that OmniSci can do this, too?

Reply:
I've added a note indicating ``D`` for Omnisci.